article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -305,7 +308,8 @@ object DotcomponentsDataModel {
       shouldHideAds = article.content.shouldHideAdverts,
       hasStoryPackage = articlePage.related.hasStoryPackage,
       hasRelated = article.content.showInRelated,
-      isCommentable = article.trail.isCommentable
+      isCommentable = article.trail.isCommentable,
+      article.metadata.commercial
```

Comment:
It might be worth only typing the parts of this we need on the Dotcomponents side when we change things there? So that we keep the interface as small as possible.

Reply:
If there's an opportunity to do this again, I'm sure it can be modelled much better than currently.  And only where the model is needed to support a feature, as @nicl says.